crates/subspace-core-primitives/src/lib.rs

```diff
@@ -461,6 +461,8 @@ impl RootBlock {
 pub type PieceOffset = u64;
 /// Piece index in consensus
 pub type PieceIndex = u64;
+/// Distance to piece index hash from farmer identity
+pub type PieceDistance = [u8; SHA256_HASH_SIZE];
```

Comment:
I don't think this belongs to core primitives, just like `PieceOffset` above actually. Both of those are implementation details and do not really exist as protocol concepts in contrast to `PieceIndex` for instance.